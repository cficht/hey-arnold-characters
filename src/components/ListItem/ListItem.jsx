import React from 'react';
import PropTypes from 'prop-types';
// import { Link } from 'react-router-dom';

const ListItem = ({ _id, name }) => (
  <>
    <p>{name}</p>
    <p>{_id}</p>
  </>
);

ListItem.propTypes = {
  _id: PropTypes.string.isRequired,
  name: PropTypes.string.isRequired
};

export default ListItem;

