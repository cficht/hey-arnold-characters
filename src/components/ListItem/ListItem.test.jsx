import React from 'react';
import { shallow } from 'enzyme';
import ListItem from './ListItem';

describe('ListItem component', () => {
  it('renders ListItem', () => {
    const wrapper = shallow(<ListItem 
      _id="5da237699734fdcb7bef8f51"
      name="Arnold Shortman" />);
    expect(wrapper).toMatchSnapshot();
  });
});
