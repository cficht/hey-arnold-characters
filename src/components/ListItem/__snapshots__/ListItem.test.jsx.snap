// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListItem component renders ListItem 1`] = `
<li>
  <Link
    to="detail/5da237699734fdcb7bef8f51"
  >
    Arnold Shortman
  </Link>
</li>
`;
